src/README.md

```diff
@@ -1,3 +1,5 @@
+This is a fork of the original at https://github.com/jss-blazor/jss-blazor updated to .NET Standard 2.1 and .NET Core 3.1
```

Comment:
Please revert these changes to the `README.md`.

Reply:
Sorry didn't mean to include this!